config/locales/cy.yml

```diff
@@ -481,77 +544,377 @@ cy:
+      body: 'Cafodd eich statws ei fŵstio gan %{name}:'
+      subject: Bŵstiodd %{name} eich statws
+      title: Hwb newydd
+  number:
+    human:
+      decimal_units:
+        format: "%n%u"
+        units:
+          billion: B
+          million: M
+          quadrillion: Q
+          thousand: K
+          trillion: T
   pagination:
+    newer: Diweddarach
     next: Nesaf
+    older: Hŷn
     prev: Blaenorol
+    truncate: "&hellip;"
   preferences:
     languages: Ieithoedd
     other: Arall
+    publishing: Cyhoeddi
     web: Gwe
+  remote_follow:
+    acct: Mewnbynnwch eich enwdefnyddiwr@parth yr ydych eisiau gweithredu ohonno
+    missing_resource: Ni ellir canfod yr URL ailgyferio angenrheidiol i'ch cyfrif
+    no_account_html: Heb gyfrif? Mae modd i chi <a href='%{sign_up_path}' target='_blank'>gofrestru yma</a>
+    proceed: Ymlaen i ddilyn
+    prompt: 'Yr ydych am ddilyn:'
+  remote_interaction:
+    proceed: Ymlaen i ryngweithio
+    prompt: 'Rydych eisiau rhyngweithio a''r tŵt hwn:'
   remote_unfollow:
     error: Gwall
     title: Teitl
+    unfollowed: Dad-ddilynwyd
   sessions:
+    activity: Gweithgaredd ddiwethaf
     browser: Porwr
     browsers:
-      alipay: ''
-      blackberry: ''
+      alipay: Alipay
+      blackberry: Blackberry
+      chrome: Chrome
+      edge: Microsoft Edge
+      electron: Electron
+      firefox: Firefox
+      generic: Porwr anhysbys
+      ie: Internet Explorer
+      micro_messenger: MicroMessenger
+      nokia: Porwr Nokia S40 Ovi
+      opera: Opera
+      otter: Otter
+      phantom_js: PhantomJS
+      qq: Porwr QQ
+      safari: Safari
+      uc_browser: UCBrowser
+      weibo: Weibo
     current_session: Sesiwn cyfredol
     description: "%{browser} ar %{platform}"
+    explanation: Dyma'r porwyr gwê sydd wedi mewngofnodi i'ch cyfrif Mastododon ar hyn o bryd.
+    ip: IP
+    platforms:
+      adobe_air: Adobe Air
+      android: Android
+      blackberry: Blackberry
+      chrome_os: ChromeOS
+      firefox_os: Firefox OS
+      ios: iOS
+      linux: Linux
+      mac: Mac
+      other: platfform anhysbys
+      windows: Windows
+      windows_mobile: Windows Mobile
+      windows_phone: Windows Phone
+    revoke: Diddymu
+    revoke_success: Sesiwn wedi ei ddiddymu yn llwyddiannus
     title: Sesiynau
+  settings:
+    authorized_apps: Apiau awdurdodedig
+    back: Yn ôl i Mastodon
+    delete: Dileu cyfrif
+    development: Datblygu
+    edit_profile: Golygu proffil
+    export: Allforio data
+    followers: Dilynwyr awdurdodedig
+    import: Mewnforio
+    migrate: Mudo cyfrif
+    notifications: Hysbysiadau
+    preferences: Dewisiadau
+    settings: Gosodiadau
+    two_factor_authentication: Awdurdodi dau-gam
+    your_apps: Eich rhaglenni
+  statuses:
+    attached:
+      description: 'Ynghlwm: %{attached}'
+      image:
+        one: "%{count} llun"
+        other: "%{count} o luniau"
+      video:
+        one: "%{count} fideo"
+        other: "%{count} fideo"
```

Comment:
Also here.
I'm sorry I can not judge whether there should be changes in singular and plural form.